docs/embedded/getting-started/containertypes.md

```diff
@@ -45,36 +52,113 @@ The following restrictions are applied to trial container types:
+
+    > [!NOTE]
+    > An Office license is not required to collaborate on Microsoft Office documents stored in a container.
 
-A standard container type is linked to a billing profile, which can be regular or direct-to-consumer billing. Each developer tenant can create up to five standard container types.
-Standard container types are created using the [New-SPOContainerType](/powershell/module/sharepoint-online/new-spocontainertype) cmdlet. For container types with regular billing, create a billing profile using the [Add-SPOContainerTypeBilling](/powershell/module/sharepoint-online/add-spocontainertypebilling) cmdlet. For container types with direct to customer billing, use the flag `-IsPassThroughBilling`. For the direct to customer billed container type, there's no need to attach a billing profile. 
+### Roles and Permissions
+
+- The admin who sets up the billing relationship for SharePoint Embedded needs to have owner or contributor permissions on the Azure subscription.
+- Admin needs to have a SharePoint Embedded Administrator or Global Admin role to operate billing cmdlets.
+
+### Azure Subscription
+
+For the Standard Billing container type, the developer admin needs to set up:
+
+- An existing SharePoint tenancy
+- An Azure subscription in the tenancy
+- A resource group attached to the Azure subscription
+
+### Standard Container Type - with billing profile
+With the standard billing profile, all consumption-based charges are directly billed to the tenant who owns or develops the application. The admin in the developer tenant  must establish a valid billing profile when creating a standard container type.
+
+![Standard](../images/1bill521.png)
+
+Each developer tenant can create up to five standard container types.
+Standard container types are created using the [New-SPOContainerType](/powershell/module/sharepoint-online/new-spocontainertype) cmdlet. 
 
 You need the following to create a standard container type:
 
 - Use SharePoint PowerShell. You must be a SharePoint Embedded Administrator or Global Administrator to run this cmdlet. If you're a SharePoint Administrator, grant yourself the SharePoint Embedded Admin role as well to execute these cmdlets.
 - An Azure subscription and a resource group must be present in the Azure portal for regular billing. 
 - An App registration must be created in Microsoft Entra ID.
 
-To create a standard container type with a regular billing profile, use the following cmdlets:
+To create a standard container type using an Azure billing profile, use the following cmdlets:
 
 ```powershell
 New-SPOContainerType [-ContainerTypeName] <String> [-OwningApplicationId] <String> [-ApplicationRedirectUrl] <String> [<CommonParameters>]
 ```
 
+Once the container type is created, add the Azure billing profile.
+
 ```powershell
 Add-SPOContainerTypeBilling –ContainerTypeId <ContainerTypeId> -AzureSubscriptionId <AzureSubscriptionId> -ResourceGroup <ResourceGroup> -Region <Region>
 ```
```

Comment:
Can you add a note here to ask the user to retry this command after a few minutes if they receive an error saying that `Microsoft.Syntex is not a registered resource provider in their subscription`? Retrying will succeed eventually.

It could go like this:

> If the cmdlet above fails with a `SubscriptionNotRegistered` error, it is because `Microsoft.Syntex` is not registered as a resource provider in the subscription. The cmdlet will send a resource provider registration request on your behalf but it will take a few minutes to be completed. Please wait 5-10 minutes and try again until the cmdlet succeeds.